api/src/opentrons/protocol_runner/legacy_command_mapper.py

```diff
@@ -65,7 +65,9 @@ def __init__(
         self._command_count: Dict[str, int] = defaultdict(lambda: 0)
 
         # equipment IDs by phyiscal location
```

Comment:
```suggestion
        # equipment IDs by physical location
```

Sneaking in this tpyo fix. 🕵️‍♂️